src/components/Employer/Signup/CompanyLogoUploader.tsx

```
@@ -0,0 +1,254 @@
+import ImageIcon from '@/assets/icons/ImageIcon.svg?react';
+import CheckIconWithFill from '@/assets/icons/CheckIconWithFill.svg?react';
+import giggleLogoPng from '@/assets/images/GiggleLogo.png';
+import AddTrigger from '@/components/Common/AddTrigger';
+import Icon from '@/components/Common/Icon';
+import InputLayout from '@/components/WorkExperience/InputLayout';
+
+const enum LogoType {
+  DEFAULT = 'default',
+  NONE = 'none',
+  SELECTED = 'selected',
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)